packages/react-composites/src/composites/CallComposite/selectors/localPreviewSelector.ts

```diff
@@ -1,17 +1,20 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { DeviceManagerState } from '@internal/calling-stateful-client';
 import * as reselect from 'reselect';
 import { getDeviceManager } from './baseSelectors';
 
-/**
- * @private
- */
-export const localPreviewSelector = reselect.createSelector([getDeviceManager], (deviceManager) => {
+const selectLocalPreview = (deviceManager: DeviceManagerState) => {

```

Comment:
We need to make it non-anonymous function so we can get function.name from this when we count for perf